src/crimson/osd/main.cc

```diff
@@ -36,9 +36,13 @@ int main(int argc, char* argv[])
                                               CEPH_ENTITY_TYPE_OSD,
                                               &cluster,
                                               &conf_file_list);
+  namespace bpo = boost::program_options;
   seastar::app_template app;
   app.add_options()
-    ("mkfs", "create a [new] data directory");
+    ("mkfs", "create a [new] data directory")
+    ("key", bpo::value<std::string>()->default_value(""), "Authentication key")
+    ("osd-uuid", bpo::value<boost::uuids::uuid>()->default_value(uuid_d().uuid), "uuid label for a new OSD")
```

Comment:
this is not correct. the options are consumed by `ceph_argparse_early_args()`, not `seastar::app_template()`. i addressed this issue in the commit named "crimson/osd: do not pass ceph options to seastar" in #26697 .